packages/trader/src/AppV2/Components/TradeTypeList/trade-type-list.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import TradeTypeListItem from './trade-type-list-item';
+import { Text } from '@deriv-com/quill-ui';
+import './trade-type-list.scss';
+
+type TradeTypeItem = {
```

Comment:
Do we have an agreement to add 'T' before type name? Or it's optional?

Reply:
I especially like the one for `error`